Jenkinsfile

```diff
@@ -17,11 +17,12 @@ pipeline {
       }
       steps {
 	sh 'python3 -c "import torch; torch.cuda.current_device()"'
+	sh 'python3 -c "import tensorflow as tf; tf.test.is_gpu_available()"'
 	sh 'python3 -m venv --system-site-packages --without-pip $HOME'
 	sh '''#!/bin/bash -ex
 	  source $HOME/bin/activate
 	  python3 setup.py develop
-	  KYMATIO_BACKEND=$STAGE_NAME python3 -m pytest --cov=kymatio
+	  KYMATIO_BACKEND=$STAGE_NAME CUDA_VISIBLE_DEVICES=0 TF_FORCE_GPU_ALLOW_GROWTH=true python3 -m pytest --cov=kymatio
```

Comment:
but so this only checks coverage for non-gpu code?

Reply:
Actually, `CUDA_VISIBLE_DEVICES` forces us to use the first GPU for the tests. The Jenkins node has two GPUs, so to prevent any shenanigans (where it allocates something on one GPU and then tries to use it on the other one), we force it to only use one.

On the other hand, it might be better to leave it unrestricted. May miss some subtle bugs by doing this restriction. At any rate, once we have multi-GPU support, we'll want to use both.

Reply:
thinking of it, I think it could be nice to remove and not to force anything, now that tf works properly